system/src/iosMain/com.splendo.kaluga/system/network/NetworkManagerBuilder.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.system.network
+
+import com.splendo.kaluga.base.IOSVersion
+
+actual class NetworkManagerBuilder : BaseNetworkManager.Builder {
+
+    override fun create(onNetworkStateChange: NetworkStateChange): BaseNetworkManager {
+        return if (IOSVersion.systemVersion > IOSVersion(12)) {
+            NetworkManager.NWPathNetworkManager(onNetworkStateChange)
```

Comment:
Isn't this available in iOS12? @avdyushin

Reply:
yes https://developer.apple.com/documentation/network/nwpathmonitor

Reply:
Then why do you only use it on 13+?

Reply:
092162a